shopfloor_reception/models/shopfloor_menu.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2025 ACSONE SA/NV (https://www.acsone.eu)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, exceptions, fields, models
+
+FILTER_TODAY_SCHEDULED_PICKINGS_HELP = """
+By default, at first step, filter the available
+pickings with the ones that are scheduled for today.
+"""
+
+
+class ShopfloorMenu(models.Model):
```

Comment:
I feel like this option is generic enough to go in shopfloor.
Could be used in other scenarios.

Reply:
@mmequignon I've put it here because today's pickings are only used in reception scenario for now.